### ISSUE TITLE
Connector for Meilisearch

### ISSUE THREAD

**Issue by Ja7ad** [estuary/connectors#1835]
### System Name

Meilisearch

### Type

Materialize

### Details

I want cdc source database (sql, nosql) to Meilisearch as index.

**Comment by dyaffe** [comment]
Hey @Ja7ad, that makes sense.  Are you set on Meilisearch?  We have requests for Algolia and already have a connector for Elastic.  I'd imagine you are, but one of those would be faster so I just wanted to confirm.

**Comment by Ja7ad** [comment]
> Hey @Ja7ad, that makes sense.  Are you set on Meilisearch?  We have requests for Algolia and already have a connector for Elastic.  I'd imagine you are, but one of those would be faster so I just wanted to confirm.

Yes I want for Meilisearch, I can write a connector for Meilisearch if you confirm it and assign it.

**Comment by dyaffe** [comment]
Hey @Ja7ad , I spoke to our team and we would be happy to have you build a connector.  Our recommendation would be to model it off of our Elastic one [here](https://github.com/estuary/connectors/tree/main/materialize-elasticsearch).  

Please feel free to reach out to our team if you have any questions.  Our [slack](https://estuary-dev.slack.com/join/shared_invite/zt-86nal6yr-VPbv~YfZE9Q~6Zl~gmZdFQ#/shared-invite/email) is likely a good place to ask any that you may have.

**Comment by Ja7ad** [comment]
> Hey @Ja7ad , I spoke to our team and we would be happy to have you build a connector.  Our recommendation would be to model it off of our Elastic one [here](https://github.com/estuary/connectors/tree/main/materialize-elasticsearch).  
> 
> Please feel free to reach out to our team if you have any questions.  Our [slack](https://estuary-dev.slack.com/join/shared_invite/zt-86nal6yr-VPbv~YfZE9Q~6Zl~gmZdFQ#/shared-invite/email) is likely a good place to ask any that you may have.

Please assign to me, I do it.
